### DISCUSSION TITLE
I am not finding option  ACTIONS after I imported the repo

### DISCUSSION THREAD

**Discussion by soundd2** [coreybutler/nvm-windows#1037] [General]
It is general question, not specfic to nvm
Please suggest

![image](https://github.com/coreybutler/nvm-windows/assets/94822959/11615e60-754b-4fdc-9090-c1cd0e3248c8)

**Comment by coreybutler** [comment]
I'm closing this because it is not related to NVM4W.

That said, you can find the info you need at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository.

**Reply by soundd2** [reply]
HI Corebutler,

Can you provide me general github support link, I am facing following issue. so that I will raise new issue over there. 



issue:

here is the my github repo, you can see 

![image](https://github.com/coreybutler/nvm-windows/assets/94822959/fcdde813-70db-49dd-bbb7-c65a2872768f)


when I tried to click on Actions--> created your own workflow --> I supposted to see code files on the left side right? but I am not seeing it.

second question is, if I write some content and commit and then you can see following error, Please suggest

![image](https://github.com/coreybutler/nvm-windows/assets/94822959/c631bef2-04d3-4137-889c-5a86a113a898)

**Reply by coreybutler** [reply]
No, this is the kind of thing you should reach out to Github about, or Google for answers. I can answer questions about this project, but I don't work for Github and I do not support their general application. The link I provided you above is the starting point for Github Actions.
